tools/quantification/quantification.xml

```diff
@@ -0,0 +1,89 @@
+    ln -s '$primary_mask' primary_mask.tiff &&
+    #if $supp_masks
+        ln -s '$supp_masks' supp_mask.tiff &&
+    #end if
+
+    mkdir ./tool_out &&
+
+    @CMD_BEGIN@
+
+    --masks primary_mask.tiff
+    #if $supp_masks
+        supp_mask.tiff
+    #end if
+
+    --image input.ome.tiff
+    --output ./tool_out
+
+    #if strip($mask_props)
+        --mask_props '$mask_props'
+    #end if
+    #if strip($intensity_props)
+        --intensity_props '$intensity_props'
+    #end if
+
+    --channel_names '$channel_names' &&
+
+    mv tool_out/*primary_mask.csv primary_mask.csv
+    ]]></command>
+
+    <inputs>
+        <param name="image" type="data" format="tiff" label="Registered TIFF"/>
+        <param name="primary_mask" type="data" format="tiff" label="Primary Cell Mask"/>
+        <param name="supp_masks" type="data" optional="true" format="tiff" label="Additional Cell Masks"/>
+        <param name="channel_names" type="data" format="csv" label="Marker Channels"/>
+        <param name="mask_props" type="text" label="Mask Metrics" help="Space separated list of additional metrics to be calculated for every mask."/>
+        <param name="intensity_props" type="text" label="Intensity Metrics" help="Space separated list of additional metrics to be calculated for every marker separately."/>
+    </inputs>
+
+    <outputs>
+        <data format="csv" name="cellmask" from_work_dir="primary_mask.csv" label="CellMaskQuant"/>
+        <collection type="list" name="quantification" label="${tool.name} on ${on_string}">
+          <discover_datasets pattern="__designation_and_ext__" format="csv" directory="tool_out/" visible="false"/>
+        </collection>
+     </outputs>
+     <tests>
+        <test>
+            <param name="image" value="test.tiff" />
+            <param name="primary_mask" value="mask.tiff" />
+            <param name="channel_names" value="channels.csv" />
+            <output name="cellmask" ftype="csv">
+                <assert_contents>
+                    <has_n_columns n="11" sep="," delta="1" />
+                </assert_contents>
+            </output>
+        </test>
+    </tests>
+    <help><![CDATA[
+    
+--------
+MCQUANT
+--------
+**MCQUANT** module for single cell quantification given a segmentation mask and multi-channel image. The CSV structure is aligned with histoCAT output.
+
+**Inputs**
+1. A fully stitched and registered image in .ome.tif format. Nextflow will use images in the registration/ and dearray/ subfolders as appropriate.
+2. One or more segmentation masks in .tif format. Nextflow will use files in the segmentation/ subfolder within the project.
+3. A .csv file containing a marker_name column specifying names of individual channels. Nextflow will look for this file in the project directory.
+
+**Outputs**
+A cell-by-feature table mapping Cell IDs to marker expression and morphological features (including x,y coordinates).
+
+# Main developer
+Denis Schapiro (https://github.com/DenisSch)
```

Comment:
are those the Galaxy tool devs? If so we have this here: https://docs.galaxyproject.org/en/latest/dev/schema.html#tool-creator-person

If not the citation should be enough and I would remove this from the tool readme.